server.js

```diff
@@ -32,18 +32,25 @@ const localesMiddleware = require('./middleware/locales');
 const { noCache } = require('./middleware/cached');
 const previewMiddleware = require('./middleware/preview');
 
-if (SENTRY_DSN) {
-    Raven.config(SENTRY_DSN, {
-        environment: appData.environment,
-        dataCallback(data) {
-            delete data.modules;
-            // clear out POST data
-            delete data.request.data;
-            return data;
-        }
-    }).install();
-    app.use(Raven.requestHandler());
-}
+/**
+ * Configure Sentry client
+ * https://docs.sentry.io/clients/node/config/
+ * https://docs.sentry.io/clients/node/usage/#disable-raven
+ */
+Raven.config(SENTRY_DSN, {
+    environment: appData.environment,
+    autoBreadcrumbs: true,
```

Comment:
Enabled `autoBreadcrumbs` to the config, notably should log any HTTP requests that the express application makes leading up to an error.